### DISCUSSION TITLE
Where are the embeddings for memory stored when memory is enabled?

### DISCUSSION THREAD

**Discussion by ShubhamVerma16** [crewAIInc/crewAI#1086] [Q&A]
While enabling memory, I want to know where are the embeddings stored and is there an option to specify the location for the database.
The code used is as followed:
`
memory=True,
        embedder={
            "provider": "cohere",
            "config":{
                "model": "embed-english-v3.0",
            "vector_dimension": 1024
            }
        },
`

The reference used is as below:
https://docs.crewai.com/core-concepts/Memory/

I want to make changes to the database manually and keep a check on the data stored in the database.

**Comment by zinyando** [comment]
According to the documentation you can customise where your storage files live by setting the `CREWAI_STORAGE_DIR`. This gives you flexibility and allows you to have things where you want them and makes it easy for you to inspect the data.

You can read more about it under this section https://docs.crewai.com/core-concepts/Memory/#implementing-memory-in-your-crew

**Comment by ShubhamVerma16** [comment]
Hi,
Thank you for the quick response. It helped me resolve the issue.
I wanted to understand how many conversation flows do i need to go through before the agents start reflecting usage of memory.

I have stored data in short term memory and want the agent to pick details from the same in the net cycle, but it does not look like it is referring the memory database.

Could you please help me with the issue.
